Doc/tools/.nitignore

```diff
@@ -50,7 +50,6 @@ Doc/library/platform.rst
 Doc/library/plistlib.rst
 Doc/library/profile.rst
 Doc/library/pyexpat.rst
-Doc/library/readline.rst
 Doc/library/resource.rst
```

Comment:
One warning remains for 3.12: `py:const reference target not found: readline.__doc__ [ref.const]`
```suggestion
Doc/library/readline.rst
Doc/library/resource.rst
```